### PR TITLE
feat:added weather feature

### PR DESCRIPTION
**Pull request by deepak29-git**
Added weather feature

Screenshot (145)

### CODE REVIEW

File: src/api-services/weather-api.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  const API_KEY = "c2922281fce40ec5f95a328c94bccb4b";
+
+  try {
+    const { data } = await axios.get(
+      `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=${API_KEY}&units=metric`
+    );
+
+    setWeather(data);
+  } catch (error) {
+    console.log(error);
```
Comment (Mohitkumar33) [inline src/api-services/weather-api.js:12]: you can do console.errror

**Review by Mohitkumar33 — APPROVED** [review]
good to merge

File: src/api-services/weather-api.js (hunk 1 above)
Comment (deepak29-git) [reply src/api-services/weather-api.js:12]: ok

**Review by tushargoyal812 — COMMENTED** [review]
code is clean and to the point
variable naming is good.
file structure is good

**Pull request merged**